demo/loadmore.html

```diff
@@ -0,0 +1,83 @@
+<html>
+
+<head>
+  <title>Load more</title>
+  <meta name="viewport" content="width=device-width, minimum-scale=1.0, initial-scale=1, user-scalable=yes">
+
+  <style>
+    body {
+      font-family: Roboto, Helvetica, sans-serif;
+      max-width: 600px;
+      height: 100vh;
+      margin: 0px auto;
+      display: flex;
+      flex-direction: column;
+    }
+
+    virtual-scroller {
+      flex: 1;
+      margin: 10px 0;
+      border-top: 2px solid #eee;
+      border-bottom: 2px solid #eee;
+    }
+
+    contact-element,
+    button {
+      height: 80px;
+    }
+  </style>
+</head>
+
+<body>
+  <div id=errors></div>
+  <div>
+    <button id=swapButton>switch virtual-scroller &lt;-&gt; div</button>
+  </div>
+
+  <virtual-scroller id=scroller></virtual-scroller>
+
+  <script type="module">
+    import * as confirmFeatures from './util/confirm-features.mjs';
+    confirmFeatures.confirm(errors);
+  </script>
+  <script type="module">
+    import 'std:elements/virtual-scroller';
+    import './contacts/contact-element.mjs';
+    import { ContactDataSource } from './contacts/contact-data-source.mjs';
+    import * as util from './util/util.mjs';
+
+    const dataSource = new ContactDataSource();
+
+    const button = document.createElement('button');
+    scroller.appendChild(button);
+
+    const loadMore = async () => {
```

Comment:
Style nit: you are using two styles of function declaration here, the `const f = () => { ... }` style here, and the `function f() { }` style below (line 74). Pick one. (I personally prefer the second style.)

Reply:
done.